dist/src/main/java/io/camunda/application/commons/service/ManagementServicesConfiguration.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application.commons.service;
+
+import io.camunda.application.commons.service.ManagementServicesConfiguration.ClientModeProperty;
+import io.camunda.application.commons.service.ManagementServicesConfiguration.LicenseKeyProperties;
+import io.camunda.service.ManagementServices;
+import io.camunda.service.license.CamundaLicense;
+import io.camunda.service.license.SaaSLicense;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties({LicenseKeyProperties.class, ClientModeProperty.class})
+public class ManagementServicesConfiguration {
+  private static final String SELF_MANAGED_MODE = "self-managed";
+  private final LicenseKeyProperties licenseKeyProperties;
+  private final ClientModeProperty clientModeProperty;
+
+  @Autowired
+  public ManagementServicesConfiguration(
+      final LicenseKeyProperties licenseKeyProperties,
+      final ClientModeProperty clientModeProperty) {
+    this.licenseKeyProperties = licenseKeyProperties;
+    this.clientModeProperty = clientModeProperty;
+  }
+
+  @Bean
+  public ManagementServices managementService(final CamundaLicense camundaLicense) {
+    final ManagementServices managementServices =
+        new ManagementServices().withLicense(camundaLicense);
+    // trigger to log if license is valid or not during startup
```

Comment:
❓ Is this a todo? Because the method below does just a check?

Reply:
It was a comment explaining what would do the logging

I removed it since I refactored how the license key is initalized, and where the logging is coming from. Its now happening when the license is being created